src/main/haskell/kore/test/Test/Kore/MetaML/Lift.hs

```diff
@@ -452,87 +452,6 @@ test_lift =
                 }
             )
         )
-    , testCase "Lift Meta Alias Declaration"
```

Comment:
Why did you delete these tests?

Reply:
The alias declarations in these tests are not syntactically-valid Kore (650ce2a32f90f975cb3a5aab787de91187c33876).